scheduler.go

```diff
@@ -878,3 +879,26 @@ func (s *Scheduler) cron(cronExpression string, withSeconds bool) *Scheduler {
 	s.setJobs(append(s.Jobs(), job))
 	return s
 }
+
+func (s *Scheduler) newJob(interval int) *Job {
+	if s.waitForInterval {
+		return newJob(interval, false)
+	}
+	return newJob(interval, true)
+}
+
+// WaitForSchedules defaults the scheduler to create all
+// new jobs with the WaitForSchedule option as true.
+// The jobs will not start immediately but rather will
+// wait until their first scheduled interval.
+func (s *Scheduler) WaitForSchedules() {
```

Comment:
I think the users might think `WaitForSchedules()` and `WaitForSchedule()` have the same behavior. Because we have other methods like `Second()`-`Seconds()`,  `Minute()`-`Minutes()` etc which in spite of being singular and plural have the same behavior. 

The comments are clear enough, but maybe a different name for `WaitForSchedules` might be nice.

Reply:
`WaitForSchedulesAllJobs()`

Reply:
Sure. Or of the word "Hold" makes sense, they can both be changed to `HoldJobTillFirstSchedule` and `HoldAllJobsTillFirstSchedule`

Reply:
We could also skip the word "First" to make it shorter :P

Reply:
how about `WaitForScheduleAll` i pushed that, let me know

Reply:
Sure 👍🏽